Assets/_Functionalities/Wms/Scripts/WMSLayerGameObject.cs

```diff
@@ -78,15 +97,50 @@ private void SetRenderOrder(int order)
             WMSProjectionLayer.RenderIndex = -order;
         }
 
+        private void HandleCredentials(StoredAuthorization auth)
```

Comment:
Deze hele functie is een duplicate van een die je in de Tile3DLayerGameObject ook hebt geimplementeerd; ik vind dit ook een gevaarlijke duplicatie omdat hier best belangrijke dingen gebeuren.

Tevens, zelfde opmerking hier als daar:

Als je dit veranderd zodat de HeaderBasedAuthorization een HeaderValue property of method heeft, dan encapsuleer je de opbouw waar die hoort - en dan hoef je hier alleen een check te doen of het een HeaderBasedAuthorization is en geen onderscheid tussen UsernamePassword en Bearer

Reply:
Hiervoor is een aparte ticket gemaakt, het is beter om het credentials object door te geven